### DISCUSSION TITLE
Pricing based custom product options

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#46] [Feature Requests]
1. I have products that are sold based on their weight. How can I set different prices for a product based on their weight. In the current version of the product, I have not found a way to do this. Therefore, I propose to create an opportunity to set prices based on certain product options.

For example:
1kg - $7
2kg - $10

2. Product is sold on a weight basis and the product has different weights. It would be nice if we could set tiered prices. based on each product weight.

For example:
1kg - 1pc = $10
1kg - 3pcs = $9
2kg - 1pc = $13
2kg - 2pcs = $18

**Comment by glennjacobs** [comment]
I think you would set the product price for 1kg for example, and then use the qty to determine how much it costs. 
E.g.

1 x 1kg = $7.00
2 x 1kg = $14.00
3 x 1kg = $21.00

Then you can use tiered pricing to provide any discounts.
e.g.

1 x 1kg = $7.00
2 x 1kg = $10.00
3 x 1kg = $12.00

**Reply by devmansurov** [reply]
That is, if now it is correct to use product variants, then such a solution is already possible so?

**Reply by glennjacobs** [reply]
If you have pre-set sizes, you could use variants, yes.
